groovy/postInit/components/Batteries.groovy

```diff
@@ -125,14 +149,46 @@ ASSEMBLER.recipeBuilder()
         .duration(200)
         .EUt(VA[LV])
         .buildAndRegister()
-
 RecyclingHelper.handleRecycling(metaitem('battery.ni_fe'), [ore('plateSteel') * 2])
 
+// Nickel-Zinc Battery
+
+ASSEMBLER.recipeBuilder()
+        .inputs(metaitem('battery.hull.mv'))
+        .inputs(metaitem('anode.zinc_oxide') * 1)
+        .inputs(metaitem('cathode.nioh2') * 1)
+        .inputs(ore('foilPlastic') * 1)
```

Comment:
why doent Ni-Fe batteries have a separator? Also I think the ebonite mv hull will dissolve in alkali.

Reply:
okay, sounds good. can we change this to a woven foil of nylon as I think that is more porous and interacts well in the aqueous environment